### PR TITLE
:bug: Source Typeform: 499 handling

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
resolved: https://github.com/airbytehq/oncall/issues/3249
499 error status code: client closed request

## How
reason: api server closed the request due to read timeout, as typeform is a certified connector read timeout is 10 minutes
added error handler with error message describing the issue.

### CODE REVIEW

**Review by lazebnyi — APPROVED** [review]

**Review by maxi297 — COMMENTED** [review]
It seems 499 means that the client unexpectedly closed the connection while the server is still processing the request. I would need more information to know if the solution makes sense. Is this occurring consistently for the user? Is it always on the same request? As 4XX are client errors, could we change something on our side to alleviate the issue? Would increasing the read timeout help?

**Comment by darynaishchenko** [comment]
> It seems 499 means that the client unexpectedly closed the connection while the server is still processing the request. I would need more information to know if the solution makes sense. Is this occurring consistently for the user? Is it always on the same request? As 4XX are client errors, could we change something on our side to alleviate the issue? Would increasing the read timeout help?

@maxi297 Yes, it's always for /responses endpoint for one user. We need to increase read timeout in this case but it's 10m for our api server. On connector side we can add handling  for this status code, Fail or Retry action on 499 status code.

**Comment by darynaishchenko** [comment]
@maxi297 Unfortunately, I don't see any other solution except handling this error. For example, previous sync in this connection was succeeded. So, adding an error messageand fail action to give a user a reason what was wrong can be helpful here. Or I switch to retry  action.  

> What's the impact of the read timeout being 10m for our api server?

My guess that the server closes the request due to timeout more than 10 minutes.

File: airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml [hunk 1]
```diff
@@ -32,6 +32,14 @@ definitions:
...
     request_parameters: {}
     request_headers: {}
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - http_codes: [499]
+              action: FAIL
+              error_message: "Source Typeform was waiting to long time for response from Typeform API, please retry later."
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml:42]: I would rephrase to something like : "Could not complete the stream: Source Typeform has been waiting for too long for a response from Typeform API. Please try again later."

**Review by maxi297 — APPROVED** [review]
Given the issue is inconsistent, we will consider this error as a server error and fail on it. Retrying might solve the issue for the user
